pywhat/Data/regex.json

```diff
@@ -1363,6 +1363,23 @@
          "Invalid": []
       }
    },
+   {
+      "Name": "Italian Fiscal Code",
+      "Regex": "^([A-Za-z]{6}[0-9]{2}[ABCDEHLMPSTabcdehlmpst]{1}[0-9]{2}[A-Za-z]{1}[0-9]{3}[A-Za-z]{1})$",
+      "plural_name": false,
+      "Description": "https://en.wikipedia.org/wiki/Italian_fiscal_code",
```

Comment:
Remove the description. If a user want to know about the code, they can Google it.